Automattic/wp-calypso
# Untangling: introduce /sites/overview/:site route

Related to:

- https://github.com/Automattic/dotcom-forge/issues/9152

## Proposed Changes

This PR adds a `/sites/overview/:site` route, which is the old `/overview/:site` route. This is so that all routes under sites dashboard have a `/sites/*` prefix consistently.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click any site
3. Verify it opens the Overview panel correctly with the new route
4. REGRESSION: Test also with `?flags=-untangling/hosting-menu`; verify it still opens the old `/overview/:site` route.

## Pre-merge Checklist
